samples/snippets/src/main/java/com/example/datastore/aggregation/CountAggregationInTransaction.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.datastore.aggregation;
+
+import com.google.cloud.datastore.AggregationQuery;
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.Datastore.TransactionCallable;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.EntityQuery;
+import com.google.cloud.datastore.Key;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.StructuredQuery.PropertyFilter;
+import com.google.cloud.datastore.aggregation.Aggregation;
+import com.google.common.collect.Iterables;
+
+public class CountAggregationInTransaction {
+
+  public static void invoke() {
+    // [START datastore_count_aggregation_query_in_transaction]
```

Comment:
The start region tag should be placed right before the import statements and the end region tag should be the last line in the file.
Kindly move them accordingly. Same comment to all the files.

Reply:
Fixed as part of https://github.com/googleapis/java-datastore/pull/871/commits/7189f80936c2aaac76204c3848646cc38872dd28.